### ISSUE TITLE
Content bug: Screenreader Support on ARIA: tab role "tbd"

### ISSUE THREAD

**Issue by Lalaluka** [mdn/content#7445]
## What page(s) did you find the problem on?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role#screen_reader_support
and
https://developer.mozilla.org/ja/docs/Web/Accessibility/ARIA/Roles/Tab_Role#screen_reader_support

## Specific page section or heading?

**_Screen reader support_**
Booth in the English and Chinese version

## What is the problem?

The Section just says: _"tbd"_
I couldn't find any PR or Issue referencing this as Work in Progress.
I think some other ARIA Sites have the same section just containing _"tbd"_ or even nothing at all.
(Another TBD: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Region_role#screen_reader_support )
(An empty Section: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Document_Role#screen_reader_support )
But I couldn't find any example of a "Screenreader support" section on an ARIA Article actually containing Infos.

## What did you expect to see?

I guess there should be sections explaining how different Screenreaders handle the ARIA roles for the Users or what the recommended implementation of Screenreaders is?
I'm not really sure what should be present instead of tbd because I couldn't find any filled example.
If someone could provide me an example I can take a look if I'm able to help fill the empty sections.

## Did you test this? If so, how?

No tests

**Comment by Ryuno-Ki** [comment]
Back in April I asked a similiar question and received a response by Chris: https://github.com/mdn/content/issues/3924#issuecomment-815559861

Sadly, it is mainly a lack of people having the time to go through the specification and adding documentation.
That being said, we're open for contributions.

@ericwbailey is topic owner for accessibility: https://github.com/mdn/content/blob/main/REVIEWING.md#topic-review-owners

Getting data could be hard to come by as not even https://a11ysupport.io/tech/aria/tab_role lists some.
You might also find other people with knowledge on the subject at https://github.com/accessibilitysupported/a11ysupport.io contributor page.

**Comment by ericwbailey** [comment]
+1 to referring to a11ysupport.io. 

I agree that TBD isn't helpful, but AT support is a little more tricky than a binary "works or doesn't." Generating, and staying on top of this sort of thing would be a massive undertaking.

@Lalaluka Would you have preferred no indication of support instead?

**Comment by Lalaluka** [comment]
Hi @ericwbailey and @Ryuno-Ki ,
thanks for the fast answer ✌️
And the provided links. They give a lot more context about the topic :)
I agree that keeping on top of accessibility information is quite a task. 

I would probably like some more indication that such information is still actively in the works.
A simple "tbd" as plaintext looks like something left over by templates and not actively maintained.

Maybe a Blockquote or Informationbox like:
> Screenreader support is still tbd visit https://github.com/mdn/content/blob/main/rfcs/aria-roles.md for informations.

Also the pages like: https://developer.mozilla.org/enUS/docs/Web/Accessibility/ARIA/Roles/Document_Role#screen_reader_support
Containing empty sections should follow the same Layout.

**Comment by Ryuno-Ki** [comment]
I filled #7561 to get a bit closer to a good solution.
Thanks for bringing it up, @Lalaluka!

**Comment by ericwbailey** [comment]
Yes @Lalaluka, thank you for this!

**Comment by Ryuno-Ki** [comment]
Okay, your proposal is in. The change should be visible at most in 48 hours from now.

That still leaves us with the question, what structure to use in general for these pages.
@ericwbailey How reliable is a11ysupport.io?
Can we have links to the relevant pages there in the first place?

I can see a data directory there with several JSON files in them.
e.g. https://github.com/accessibilitysupported/a11ysupport.io/blob/8bc28f0f25cbdee54acaa6f0ee90d69c60cbbfa2/data/tech/aria/tab_role.json

I'd prefer to consume an API (which we could then pull into Yari) instead of going with raw GitHub files.
https://www.npmjs.com/package/a11ysupport doesn't look official enough to me.

Is it something you think I could discuss over there in their repo?

It's like, I would like to point people to experts (and perhaps Open Web Docs Foundation could be convinced to support them in some way) instead of having to double the efforts of having reliable and up-to-date information.

**Comment by ericwbailey** [comment]
>@ericwbailey How reliable is a11ysupport.io?

I have a high degree of trust, but I believe it is a one person endeavor. 

>Is it something you think I could discuss over there in their repo?

I think that would probably be the best approach, just so they know what's going on and can express what works best for their plans for the site.

>It's like, I would like to point people to experts (and perhaps Open Web Docs Foundation could be convinced to support them in some way) instead of having to double the efforts of having reliable and up-to-date information.

I can't speak for the maintainer, but I would love to see funding around this sort of thing!

**Comment by Ryuno-Ki** [comment]
I've filled https://github.com/accessibilitysupported/a11ysupport.io/issues/179.
Now let's wait and see.

**Comment by Ryuno-Ki** [comment]
@ericwbailey Do you know someone from https://www.w3.org/community/aria-at/participants ?
a11ysupport is currently a one-man-show hoping to get adopted from the W3C Community Group from what I've read today.

I will open a discussion with mdn/yari (not today; too tired from work) to explore, what is required to consume https://www.npmjs.com/package/a11ysupport similar to BCD data.

**Comment by ericwbailey** [comment]
@Ryuno-Ki I know a couple members. I have a meeting Wednesday night with someone who could point us in the right direction, will update when I get some names to consider.

>hoping to get adopted from the W3C Community Group

This would be wonderful!

**Comment by Ryuno-Ki** [comment]
@ericwbailey Any updates?
How was the call?

**Comment by ericwbailey** [comment]
@Ryuno-Ki It slipped my mind 🙃

I asked yesterday via a mutual Slack we're in, waiting for a reply.

**Comment by Ryuno-Ki** [comment]
@ericwbailey Any updates?

**Comment by bsmth** [comment]
Hi @Lalaluka thanks for raising this issue. It looks like work has been completed to fill these sections with content in multiple PRs linked above.

@Ryuno-Ki for your question:

>what structure to use in general for these pages.

This is a great topic for [a GitHub discussion](https://github.com/mdn/mdn-community/discussions) if it's still relevant.

If you think I've made a mistake, feel free to comment on this issue! Thank you :)
